website/content/docs/security/acl/acl-policies.mdx

```diff
@@ -0,0 +1,435 @@
+The following examples show the same policy formatted in HCL and JSON:
+
+<CodeTabs heading="Example rules">
+<CodeBlockConfig lineNumbers>
+
+```hcl
+# These control access to the key/value store.
+key_prefix "" {
+  policy = "read"
+}
+key_prefix "foo/" {
+  policy = "write"
+}
+key_prefix "foo/private/" {
+  policy = "deny"
+}
+# Or for exact key matches
+key "foo/bar/secret" {
+  policy = "deny"
+}
+
+# This controls access to cluster-wide Consul operator information.
+operator = "read"
+```
+
+</CodeBlockConfig>
+<CodeBlockConfig lineNumbers>
+
+```json
+{
+  "key": [
+    {
+      "foo/bar/secret": [
+        {
+          "policy": "deny"
+        }
+      ]
+    }
+  ],
+  "key_prefix": [
+    {
+      "": [
+        {
+          "policy": "read"
+        }
+      ]
+    },
+    {
+      "foo/": [
+        {
+          "policy": "write"
+        }
+      ]
+    },
+    {
+      "foo/private/": [
+        {
+          "policy": "deny"
+        }
+      ]
+    }
+  ],
+  "operator": "read"
+}
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+## Rule Scope
+
+The rules from all policies, including roles and service identities, linked with a token are combined to form that token's effective rule set.
+Policy rules can be defined in either an `allowlist` or `denylist` mode, depending on the configuration of the [`acl_default_policy`](/docs/agent/options#acl_default_policy).
+If the default policy is configured to deny access to all resources, then policy rules can be set to `allowlist` access to specific resources.
+If the default policy is to allow then access to all resources, policy rules can be used to explicitly deny access to resources.
+
+## Implementing Policies
+
+After defining policies, the person responsible for administrating ACLs in your organization can implement them through the command line or by calling the ACL HTTP API endpoint and including rules in the payload.
+
+### Command Line
+
+Use the `consul acl policy` command to manage policies. Refer to the [ACL command line documentation](/commands/acl/policy) for details.
+
+The following example creates a policy called `my-app-policy` and applies the rules defined in `rules.hcl`:
+
+```shell-session
+$ consul acl policy create -name "my-app-policy" -description "Human-readable description of my policy" -rules @rules.hcl -token "<token with ACL 'write' access>"
+```
+
+Note that the command must present a token with permissions to use the ACL system. If the command is issued successfully, the console wil print information about the policy:
+
+```shell-session
+ID:           <auto-generated ID>
+Name:         my-app-policy
+Description:  Human-readable description of my policy
+Datacenters:  <specified with the -valid-datacenter option>
+Rules:
+<rules defined in the rules.hcl file>
+```
+
+You can can define several attributes that attach additional metadata and specify the scope of the policy. See [Policy Attributes](#policy-attributes) for details.
+
+### HTTP API Endpoint
```

Comment:
I still think it's better to link to the API docs, than duplicate the API docs here.   Including examples on the API docs page would be great.